pkgs/top-level/all-packages.nix

```diff
@@ -7469,10 +7472,6 @@ with pkgs;
 
   zzuf = callPackage ../tools/security/zzuf { };
 
-  ### DEVELOPMENT / EMSCRIPTEN
-
-  buildEmscriptenPackage = callPackage ../development/em-modules/generic { };
```

Comment:
This is a substantial, breaking change and should also be explained in the release notes for 23.11.